src/NativeBridge/DataViewInterop.h

```diff
@@ -17,7 +17,12 @@ typedef MANAGED_CALLBACK_PTR(bool, GETLABELS)(DataSourceBlock *source, int col,
 // REVIEW: boost_python is not updated at the same speed as swig or pybind11.
 // Both have a larger audience now, see about pybind11 https://github.com/davisking/dlib/issues/293
 // It handles csr_matrix: https://pybind11-rtdtest.readthedocs.io/en/stable/advanced.html#transparent-conversion-of-dense-and-sparse-eigen-data-types.
-using namespace boost::python;
+using namespace pybind11;
+#if !defined(extract_or_cast)
```

Comment:
If boost_python is removed, you can probably replace extract_or_cast and has_key_or_contains everywhere.